.github/scripts/generate-lighthouse-summary.sh

```
@@ -0,0 +1,85 @@
+    SI=$(jq -r '(.audits["speed-index"].numericValue // 0) | floor' "$REPORT_FILE")
+    TTI=$(jq -r '(.audits.interactive.numericValue // 0) | floor' "$REPORT_FILE")
+
+    # Determine pass/fail for metrics
+    FCP_STATUS=$([ "$FCP" -le 2000 ] && echo "✅" || echo "❌")
+    LCP_STATUS=$([ "$LCP" -le 2500 ] && echo "✅" || echo "❌")
+    CLS_STATUS=$(awk -v cls="$CLS" 'BEGIN {print (cls <= 0.1) ? "✅" : "❌"}')
+    TBT_STATUS=$([ "$TBT" -le 300 ] && echo "✅" || echo "❌")
+    SI_STATUS=$([ "$SI" -le 3000 ] && echo "✅" || echo "❌")
+    TTI_STATUS=$([ "$TTI" -le 3500 ] && echo "✅" || echo "❌")
+
+    echo "| Metric | Value | Status | Budget |" >> "$GITHUB_STEP_SUMMARY"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)